Editor/UI/Windows/CourseWindowFixedOverlay.cs

```diff
@@ -0,0 +1,50 @@
+/* Copyright (C) Innoactive GmbH - All Rights Reserved
+ * Unauthorized copying of this file, via any medium is strictly prohibited
+ * Proprietary and confidential
+ * Innoactive GmbH, November 2020
+ */
+
+using Innoactive.CreatorEditor.UI;
+using Innoactive.CreatorPro.Account;
+using UnityEditor;
+using UnityEngine;
+using Innoactive.CreatorEditor;
+
+
+namespace Innoactive.Creator.Core
+{
+    internal class CourseWindowFixedOverlay : IEditorGraphicDrawer
+    {
+        public int Priority { get; } = 100;
+
+        private Texture2D backgroundTexture;
+
+        public CourseWindowFixedOverlay()
+        {
+            backgroundTexture = new Texture2D(1, 1);
+            backgroundTexture.SetPixel(0, 0, new Color(0, 0, 0, 0.3f));
+            backgroundTexture.Apply();
+        }
+
+        public void Draw(Rect windowRect)
+        {
+            IChapter c = GlobalEditorHandler.GetCurrentChapter();
```

Comment:
```suggestion
            IChapter chapter = GlobalEditorHandler.GetCurrentChapter();
```

[Coding Conventions](https://github.com/Innoactive/Creator/blob/develop/.github/CODING_CONVENTIONS.md): Proper variable names. :)